pkg/virt-operator/resource/apply/rbac.go

```diff
@@ -4,6 +4,7 @@ import (
 	"context"
```

Comment:
> - Add a semantic deep equal on role and clusterrole `rules` object.

You should amend the commit message since you now rolled your own comparison func.

Reply:
Added description to both commits.
Thanks!